src/db.rs

```diff
@@ -318,8 +319,16 @@ impl KbsDb {
             .bind(keysetid)
             .fetch_one(&self.dbpool)
             .await?;
-        let rks: Vec<String> = serde_json::from_str(&keyset_row.try_get::<String, _>(0)?)?;
-        Ok(rks)
+        match serde_json::from_str(&keyset_row.try_get::<String, _>(0)?) {
```

Comment:
so this match catches errors of the json parser.
it won't catch the error returned by `keyset_row.try_get`.

are we sure this is what we want?